foundations/html_css/html-foundations/working-with-text.md

```diff
@@ -158,6 +158,16 @@ Writing an HTML comment is simple: We just enclose the comment with `<!--` and `
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
+<div class="lesson-note" markdown="1">
+
+ **Note: VSCode shortcut for comments** 
+
+Let's be honest, constantly typing `<!--` and `-->` tags on your keyboard takes too long. So, here are some keyboard shortcuts you can use to save you time and ultimately a headache:
+
+1. **Mac Users:** <kbd>CMD</kbd> + <kbd>/</kbd> 
+2. **Windows and Linux Users**: <kbd>CRTL</kbd> + <kbd>/</kbd>
```

Comment:
I am a big fan of adding this keyboard shortcut, so I greatly appreciate this contribution! 

However, I think the wording is too friendly and not as informative as it could be. In addition, I do not think we need to numbering or use bold for the shortcuts. In addition, I think "Note" is not needed in the heading because the surrounding styling suggests that it is a note. So I am thinking something along these lines (but feel free to adjust it):

```suggestion
 **VSCode keyboard shortcut** 

The following shortcut will create a new comment, convert any line to a comment, or uncomment any line:

- Mac Users: <kbd>CMD</kbd> + <kbd>/</kbd> 
- Windows and Linux Users: <kbd>CRTL</kbd> + <kbd>/</kbd>
```

Reply:
How does this sound @rlmoser99:
"If you find typing out comments, the following shortcut will quickly create a new comment, convert any line to a comment, or uncomment any line"?

Reply:
I think it is missing the word "syntax".  

> If you find typing out comment syntax tiring